core/src/main/java/com/backblaze/b2/json/B2JsonObjectHandler.java

```diff
@@ -486,7 +486,7 @@ public T deserializeFromUrlParameterMap(Map<String, String> parameterMap, B2Json
                     throw new B2JsonException("required field " + fieldInfo.getName() + " cannot be null");
                 }
                 constructorArgs[fieldInfo.constructorArgIndex] = value;
-                foundFieldBits |= fieldInfo.bit;
+                foundFieldBits.set(fieldInfo.constructorArgIndex);
```

Comment:
Alex, ab and I had discussed that it seems like this code sets the bits but doesn't actually use them (unlike in deserialize(). Could you look through and see if there's some missing _usage_ of the bits and add as needed while you're in here?